issuance/cert.go

```diff
@@ -193,13 +193,16 @@ func (i *Issuer) requestValid(clk clock.Clock, prof *Profile, req *IssuanceReque
 }
 
 func (i *Issuer) generateTemplate() *x509.Certificate {
+	x509OID, _ := x509.OIDFromInts([]uint64{2, 23, 140, 1, 2, 1})
```

Comment:
It's very incongruous that you check for the error condition in both tests but not here, in the real production code. This needs to happen in such a way that the error return is checked. Perhaps the best way to accomplish that is to declare this variable at the package level, initialize it in an `init()` func, and have that panic if the construction fails.

Reply:
How about a package level with a `mustOIDFromInts` function? Does this look good to you?

```go
var domainValidatedOID = mustOIDFromInts([]uint64{2, 23, 140, 1, 2, 1})

func mustOIDFromInts(oid []uint64) x509.OID {
	x509OID, err := x509.OIDFromInts(oid)
	if err != nil {
		panic(fmt.Errorf("failed to create OID using ints %v: %w", oid, err))
	}
	return x509OID
}
```